Makefile

```diff
@@ -312,6 +312,17 @@ tidy: tidy-js tidy-perl
 test-containers-compose:
 	tools/test_containers_compose
 
+.PHONY: test-helm-chart
+test-helm-chart: test-helm-lint test-helm-install
```

Comment:
this could be called as part of https://github.com/os-autoinst/openQA/pull/4650/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R151?

Reply:
The workflow has two jobs, one with `make test-helm-lint`, the second one with `make test-helm-install`. I just call the sub-targets in parallel and I am not using the `test-helm-chart` target directly.

Reply:
Yes, I can see that the targets are used. It just would be great if we have something like a simple `make test` that simply calls every test. Do you think this can be done?

Reply:
I inspired myself in the https://github.com/os-autoinst/openQA/blob/master/.github/workflows/check-containers.yml where the same approach is used.